apps/src/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContact.jsx

```diff
@@ -147,14 +158,6 @@ export class RegionalPartnerMiniContact extends React.Component {
             defaultValue={this.state.name}
             style={style}
           />
-          {this.state.errors.includes('email') && (
```

Comment:
I also moved the error messages to be below their respective text boxes to match our error messaging elsewhere on the site.